django/contrib/contenttypes/fields.py

```diff
@@ -226,8 +224,8 @@ def __get__(self, instance, cls=None):
         pk_val = getattr(instance, self.fk_field)
 
         try:
-            rel_obj = getattr(instance, self.cache_attr)
-        except AttributeError:
+            rel_obj = self.get_cached_value(instance)
```

Comment:
Maybe `get_cached_value` should take an optional second `default` arg (like `dict.get()` does) for cases like this?